src/path.rs

```diff
@@ -389,6 +425,10 @@ pub mod parsing {
         ///     }
         /// }
         /// ```
+        #[cfg_attr(
+            syn_enable_doc_cfg,
+            doc(cfg(all(any(feature = "derive", feature = "full"), feature = "parsing")))
+        )]
```

Comment:
Like the `Attribute` methods, we don't need to copy cfgs from the struct itself.